index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+
+  <section id="header">
+    <nav class="navbar">
+      <div>
+        <img src="image/My_Logo.png" alt="My Logo">
```

Comment:
Good job having a logo. You can have it be your actual logo or just your name instead of `my logo`.

- [ ] Please change it your name :+1: and wrap it around an `anchor tag` so that it becomes clickable :100: 

The anchor tag is the `a` tag we use to put links in
```html
<a href="Your-site-link" >
 <img src="image/My_Logo.png" alt="My Logo">
</a>
```